vaadin-list-mixin.html

```diff
@@ -208,8 +208,12 @@
       return -1;
     }
 
+    _isItemHidden(item) {
+      return getComputedStyle(item).getPropertyValue('display') === 'none';
```

Comment:
You can shorten this to `getComputedStyle(item).display`

Reply:
done